freqtrade/templates/sample_strategy.py

```diff
@@ -58,6 +58,8 @@ class SampleStrategy(IStrategy):
     # Hyperoptable parameters
     buy_rsi = IntParameter(low=1, high=50, default=30, space='buy', optimize=True, load=True)
     sell_rsi = IntParameter(low=50, high=100, default=70, space='sell', optimize=True, load=True)
+    short_rsi = IntParameter(low=51, high=100, default=70, space='sell', optimize=True, load=True)
+    exit_short_rsi = IntParameter(low=1, high=50, default=30, space='buy', optimize=True, load=True)
```

Comment:
Please leave the "sample_strategy" alone (keep it as long-only strategy).

instead, add a "sample_short_strategy" (naming up for discussion). 
This strategy is intended to show people how to create a strategy and serve as a sample - and i doubt everyone will want a long/short strategy (maybe because the exchange/markets people using are long only).

Reply:
What about just commenting out the short parts? I made a `sample_short_strategy` and 99% of the code was the same, so it seems like unneccessary effort for anyone to read the `sample_short_strategy` file as well

Reply:
if i'm a new user looking for a long only strategy sample / template- i'll not want to read through however many lines that are not relevant to my goal at all.
It's however unclear if that must include 200 new lines - or 3 new lines.

can we for the moment simply leave the templates folder untouched - and work on that last? (it's not that important at this stage, and work there now is basically just "legwork" - not the complicated stuff).


the "sample's" we're looking at now are (in theory) only generated from the .j2 templates (using `freqtrade new-strategy ...`).
We'll see how much / little the effort / difference will be once the interface itself is finished / as discussed in the issue (which i can't see in this PR at all yet, but was expecting from it (at least the column namings including compat code for that).